Libplanet/Action/ActionTypeAttribute.cs

```diff
@@ -40,11 +41,20 @@ public ActionTypeAttribute(string typeIdentifier)
         /// <c>null</c>.</returns>
         public static string ValueOf(Type actionType)
         {
-            return actionType
+            var typeIdentifier = actionType
```

Comment:
it would be better if the type of `typeIdentifier` is specified. we specify variable type unless the type of the variable is self-evident.

Reply:
I added type and amended the commit. Please take a look @longfin.